From b9f10a50d60210b1459cf2e446f6d34d3dc9e178 Mon Sep 17 00:00:00 2001
From: Huang Shijie <b32955@freescale.com>
Date: Fri, 16 Dec 2011 16:27:03 +0800
Subject: [PATCH 0765/2463] ENGR00170465-1 MX6Q/SDMA : update the SDMA script
 for UART RX

The old script does not clear the IDLE flag.
So when the IDDMAEN(UCR4[6]) is enabled, the RX only receives few
bytes(such as 2 or 1 byte) per DMA operation.

The new script fixes it.

Signed-off-by: Huang Shijie <b32955@freescale.com>
---
 firmware/imx/sdma/sdma-imx6q-to1.bin.ihex |   20 ++++++++++----------
 1 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/firmware/imx/sdma/sdma-imx6q-to1.bin.ihex b/firmware/imx/sdma/sdma-imx6q-to1.bin.ihex
index 04ae992..f9ef594 100644
--- a/firmware/imx/sdma/sdma-imx6q-to1.bin.ihex
+++ b/firmware/imx/sdma/sdma-imx6q-to1.bin.ihex
@@ -1,5 +1,5 @@
 :1000000053444D4101000000010000001C000000AD
-:1000100023000000A80000006A0500008202000022
+:1000100023000000A8000000780500008202000014
 :10002000FFFFFFFF00000000FFFFFFFFFFFFFFFFDC
 :10003000FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFD0
 :10004000FFFFFFFFB61800006A1A00004B04000013
@@ -87,13 +87,13 @@
 :10056000097C286A077F0000DB57004D057DFAC132
 :10057000DB571B9A77C254040AC2189AE3C1DB57AF
 :10058000F352056AFB568E02941AC36AC862690266
-:100590001D7D941EC36ED36EC8624802C86A94263D
-:1005A000C36E981EC36EC8629826C36E6002097C33
-:1005B000C8626E02247D096A1E7F0125004D257DDB
-:1005C0007D9A286A187F04627AC2B19AE36E8F001E
-:1005D000D805017D8D00A005C8626E02107D096AF4
-:1005E0000A7F0120F97C286A067F0000004D0D7DFE
-:1005F000FAC1DB576E9A070004620C6AAE9A286A49
-:10060000FA7F04627AC258045404286AF47F0AC24A
-:020610006B9AE3
+:10059000247D941EC36ED36EC8624802C86A942636
+:1005A000981EC36ED36EC8624C02C86A9826C36E8A
+:1005B000981EC36EC8629826C36E6002097CC8622A
+:1005C0006E02247D096A1E7F0125004D257D849AD7
+:1005D000286A187F04627AC2B89AE36E8F00D80541
+:1005E000017D8D00A005C8626E02107D096A0A7F38
+:1005F0000120F97C286A067F0000004D0D7DFAC1BC
+:10060000DB576E9A070004620C6AB59A286AFA7F73
+:1006100004627AC258045404286AF47F0AC26B9AAE
 :00000001FF
-- 
1.7.7.4

